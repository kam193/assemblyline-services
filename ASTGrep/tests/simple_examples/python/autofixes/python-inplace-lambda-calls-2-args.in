(lambda s,c:B.join(B**c for B in s))([130,173,227,166,177,177,172,177,227,172,160,160,182,177,177,166,167,249,227,230,176], 3)

(lambda a,b,c:a+b+b)(1,2,3)
