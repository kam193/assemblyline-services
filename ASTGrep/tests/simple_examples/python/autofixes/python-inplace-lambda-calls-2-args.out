B.join(B**3 for B in [130,173,227,166,177,177,172,177,227,172,160,160,182,177,177,166,167,249,227,230,176])

(lambda a,b,c:a+b+b)(1,2,3)
